onboarding/mentors.md

```diff
@@ -3,4 +3,79 @@ title: Onboarding Responsibilities for Mentors of New Hires
 description: How to be an effective mentor of a new hire
 ---
 
-TBD
+# Mentorship at Artsy
+
+## Why Mentor
+One of Artsy's core values is that [People are Paramount](#). As a company we try to meet this in a number of ways, from all the work of People Ops to Ongoing development conversations between employees and managers. As an engineering team we also try to reflect this value by encouraging mentoring among peers.
+
+What does this look like in practice? In an ideal world it would be automatic and spontaneous between every member of the team. Some of our [engineering principles](https://github.com/artsy/README/blob/master/culture/engineering-principles.md#readme) appeal explicitly to this ideal - **Psychological Safety**, **Leverage Your Impact**, **De-silo Engineers** and **Being Nice is Nice** stand out - while others like **Open Source by Default** and **Incremental Revolution**, while technical on the surface have clear analogues in how we work together. In practice we understand that all of these ideals require intentional work and iteration between people to make them reality. Artsy mentors, assigned when a new engineer joins the team, work to bridge that gap.
+
+## What makes a good mentor
+The role of a mentor is purposely different from that of a boss, a manager or a technical lead, and that role shifts to accommodate the needs of a team member. There may be multiple overlapping descriptions - a mentor may be someone who
+- *Facilitates onboarding*: Simply someone who can help a new hire smooth out the bumps of getting onboarded.
+- *Knows how things work*: From a non-judgemental sounding board for ['stupid' questions](#simple-answers-to-common-questions) to someone who can simply help find the best point person or stakeholder for a project.
+- *Helps develop people skills*: Artsy's engineering and company cultures are specific and open, but there is always room for improvement and new perspectives, especially the team changes. A mentor can act as a conduit between their mentee and the team as a whole.
```

Comment:
Curious what you mean by this point! (Do you have an example?)

Reply:
I wonder if the example can be framed as highlighting successes?

Reply:
I removed this point since it felt very redundant, but followups to this document are great.